src/qml/QFieldCloudPopup.qml

```diff
@@ -238,31 +238,53 @@ Popup {
 
         Text {
           id: transferErrorText
+
+          property string errorString: '[QF/invalid] This is an invalid error'
+          property bool showAllErrorString: false
+
           visible: false
-          font: Theme.defaultFont
-          text: ''
+          font: Theme.tipFont
+          text: {
```

Comment:
NOTE this is not the only place that we show errors. Probably it's worth having a very simple component wrapping this logic.

`CloudErrorString {
  error: "error goes here"
}
`

Reply:
Oh alright, here you are, a cool collapsible message widget :)

Reply:
Yes!